modules/ROOT/pages/manage-audit-logs.adoc

```diff
@@ -47,100 +47,107 @@ You can alter the configuration per Sync Gateway database by changing the databa
 
 Below is an example of a simple database configuration.
 
-[source, json, indent=2]
+[source, json]
 ----
 {
-    "bucket": "todo", 
-    "num_index_replicas": 0
-    “logging”: {
-		         “audit”: {
- 		                    "enabled”: false,
-                        “enabled_events”: [],
-                        “disabled_users”: [
-                                            {
-		                                          “name”: “user1”,
- 	                                            “domain”: “sgw”
-                                            }
-                                          ]
-                      },
-    “disabled_roles”: [
-                        {
-		                      “name”: “role1”,
- 	                        “domain”: “sgw”
-                        }
-                      ]
-                }
+  "bucket": "todo", 
+  "num_index_replicas": 0,
+  “logging”: {
+               “audit”: {
+                          "enabled”: false,
+                          “enabled_events”: [],
+                          “disabled_users”: [
+                                              {
+                                                “name”: “user1”,
+                                                “domain”: “sgw”
+                                              }
+                                            ],
+                        },
+  “disabled_roles”: [
+                      {
+                        “name”: “role1”,
+                        “domain”: “sgw”
+                      }
+                    ],
+               },
```

Comment:
This json isn't valid, can you pop this into a validator?

1. there is a trailing comma on the last element
2. there are smart quotes on the keys
3. there's missing a `}` to close `"logging"`

Reply:
Done. Forgot how fiddly JSON validation was!